backend/infrahub/core/schema/definitions/core/webhook.py

```diff
@@ -12,11 +9,48 @@
     "order_by": ["name__value"],
     "display_labels": ["name__value"],
     "include_in_menu": False,
+    "icon": "mdi:webhook",
     "branch": BranchSupportType.AGNOSTIC.value,
     "uniqueness_constraints": [["name__value"]],
     "attributes": [
         {"name": "name", "kind": "Text", "unique": True, "order_weight": 1000},
-        {"name": "description", "kind": "Text", "optional": True, "order_weight": 2000},
+        {
+            "name": "event_type",
+            "kind": "Text",
+            "enum": ["all"] + EventType.available_types(),
+            "default_value": "all",
+            "order_weight": 1500,
+            "description": "The event type that triggers the webhook",
```

Comment:
It's not possible today but I think it would be nice to have a select type attribute where you'd be able to select one or more entries from `["all"] + EventType.available_types()`

Reply:
Agreed, I wish we could do that too